pkg/sentry/server/server.go

```diff
@@ -44,17 +44,21 @@ type Options struct {
 	// Security is the security handler for the server.
 	Security security.Handler
 
-	// Validator is the client authentication validator.
-	Validator validator.Validator
+	// Validator are the client authentication validator.
+	Validators map[sentryv1pb.SignCertificateRequest_TokenValidator]validator.Validator
+
+	// Name of the default validator to use if the request doesn't specify one.
+	DefaultValidator sentryv1pb.SignCertificateRequest_TokenValidator
 
 	// CA is the certificate authority which signs client certificates.
 	CA ca.Signer
 }
 
 // server is the gRPC server for the Sentry service.
 type server struct {
-	val validator.Validator
-	ca  ca.Signer
+	vals             map[sentryv1pb.SignCertificateRequest_TokenValidator]validator.Validator
```

Comment:
```suggestion
	validators             map[sentryv1pb.SignCertificateRequest_TokenValidator]validator.Validator
```

`vals` is a bit of an overloaded word. I would think it is a slice integers or something.

Reply:
Done